### DISCUSSION TITLE
Migrate to Bun ?

### DISCUSSION THREAD

**Discussion by Enzeekay** [anse-app/anse#99] [Ideas]
I have seen the plan to migrate to rust in the project, it sound really good. However, Bun is taking the new roadmaps by storm, is it a viable option to migrate into Bun? 

Vercel already supported Bun installation : https://vercel.com/changelog/bun-install-is-now-supported-with-zero-configuration

Bun support reading .env by default, which is quite convenience for this project already

**Comment by ddiu8081** [comment]
I'm interested in this, before that I'm going to test bun+Astro on other projects first. 👋
